modules/ROOT/pages/organization-management-in-bonita-bpm-studio.adoc

```diff
@@ -1,7 +1,7 @@
 = Organization management in Bonita Studio
 :description: Organization is the set of users, groups and roles. It is used indirectly to define privileges for tasks execution, applications, API, etc.
 
-Organization is the set of users, groups and roles. It is used indirectly to define privileges for tasks execution, applications, API, etc.
+{description}
 
 Within Bonita Studio, more than one organization can be defined. This is useful if you are developing a process that will be deployed in several different organizations. It is also useful for testing, because you can create a reduced version of your organization that has the same structure but fewer users, saving time in managing your test environment.
```

Comment:
in several different organizations. => in several organizations.

Reply:
Done